docs/install/harvester-configuration.md

```diff
@@ -75,10 +75,18 @@ install:
   vip_mode: dhcp
   force_mbr: false
   addons:
-    rancher-monitoring:
+    harvester_vm_import_controller:
```

Comment:
@w13915984028 Please help check this part. Thanks!

Reply:
need to double check:

IIRC, string like rancher_monitoring in the YAML will be converted to rancherMonitoring in bootstrap stage.